### DISCUSSION TITLE
WorldWindJava community edition 2.2

### DISCUSSION THREAD

**Discussion by jrepko** [WorldWindEarth/WorldWindKotlin#75] [General]
I have a java project that was using the community edition of worldwindjava from maven central.   Today, my builds no longer work because the artifact appears to be missing.

Is there some new thing to use or a migration path?  Or do I have to make my own jar locally and abandon the maven repository?

**Comment by EMaksymenko** [comment]
What repository did you use?

**Reply by jrepko** [reply]
https://repo.maven.apache.org/maven2/
and
https://jcenter.bintray.com/

I just noticed the posting on Bintray about officially finally going away.  My suspicion is that the artifact was coming from there and once it disappeared, it had to rely on Maven Central.

**Reply by EMaksymenko** [reply]
There is only official NASA Java lib available there.
We are currently not maintaining any legacy WorldWind versions releases except Kotlin.
So you will need to build a JAR.
